books/free-programming-books-langs.md

```diff
@@ -2168,6 +2168,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [A Gentle Introduction To Rust](https://stevedonovan.github.io/rust-gentle-intro) - Steve J Donovan
 * [Asynchronous Programming in Rust](https://rust-lang.github.io/async-book)
 * [Easy Rust](https://dhghomon.github.io/easy_rust/) - David McLeod (HTML, PDF)
+* [From JavaScript to Rust ebook](https://github.com/wasmflow/node-to-rust) - Jarrod Overson
```

Comment:
```suggestion
* [From JavaScript to Rust ebook](https://github.com/wasmflow/node-to-rust/releases/download/v1/from-javascript-to-rust.pdf) - Jarrod Overson
```

Maybe it's better to use this link, the README can be confusing

Reply:
And what about pointing to current HEAD instead of a released tag?

https://github.com/wasmflow/node-to-rust/raw/HEAD/from-javascript-to-rust.pdf

Reply:
you're right, I didn't think about that